src/tests/ftest/daos_test/daos_core_test-rebuild.py

```diff
@@ -32,6 +32,9 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [["DAOS-6442", "test_name", "rebuild tests 18"]]
```

Comment:
While we're making changes can we also fix https://github.com/daos-stack/daos/blob/master/src/tests/ftest/daos_test/daos_core_test-rebuild.yaml#L60:
```suggestion
    CANCEL_FOR_TICKET = [
        ["DAOS-5851", "test_name", "rebuild tests 0-10"]
        ["DAOS-6442", "test_name", "rebuild tests 18"]
    ]
```
+ uncomment the `daos_test/daos_core_test-rebuild.yaml` variant?